homeassistant/components/homewizard/strings.json

```diff
@@ -1,5 +1,6 @@
 {
   "config": {
+    "flow_title": "{name}",
```

Comment:
This is now the default, and should not be needed anymore.

Reply:
Also with `"name": f"{self.config[CONF_PRODUCT_NAME]} ({self.config[CONF_SERIAL]})"`?

Reply:
<img width="287" alt="Screenshot 2022-01-27 at 13 57 59" src="https://user-images.githubusercontent.com/74970928/151363420-6c1c0d88-5b39-432a-a69f-ff1c9d16d7aa.png">

vs "HomeWizard Energy"

Reply:
yes, the frontend does the magic as of this release:

<https://github.com/home-assistant/frontend/pull/11358>

Reply:
I see, `self.context` is needed but `flow_title` is now handled. Nice :)!